nbcelltests/test.py

```diff
@@ -69,17 +69,17 @@ def assemble_code(notebook):
         if _is_empty(cell['source']):
 
             skiptest = "@nbcelltests.tests_vendored.unittest.skip('empty code cell')\n" + INDENT
-        elif _is_empty("".join(test_lines).replace(r"%cell", "pass")):
+        elif _is_empty("".join(test_lines).replace(r"%cell", "pass # no test was supplied")):
             skiptest = "@nbcelltests.tests_vendored.unittest.skip('no test supplied')\n" + INDENT
         elif not _cell_source_is_injected(test_lines):
             skiptest = "@nbcelltests.tests_vendored.unittest.skip('cell code not injected into test')\n" + INDENT
         else:
             skiptest = ""
 
-        cells.append([i, [], "%sdef test_code_cell_%d(self):\n" % (skiptest, code_cell)])
+        cells.append([code_cell, [], "%sdef test_code_cell_%d(self):\n" % (skiptest, code_cell)])
```

Comment:
For a future cleanup: This is somewhere where a namedtuple would be nice.

Reply:
Ha ha, almost anything would be better than what's here :)

Reply:
@vidartf you could probably have added this comment as a TODO into the code while doing the review. I'll add this one before merging, though.

Reply:
This should make it easier:
```suggestion
        # TODO: Use namedtuples for these:
        cells.append([code_cell, [], "%sdef test_code_cell_%d(self):\n" % (skiptest, code_cell)])
```